src/components/visualizations/utils/test/drilldownEventing.spec.tsx

```diff
@@ -506,4 +506,105 @@ describe("Drilldown Eventing", () => {
             expect(drillConfig.onFiredDrillEvent).toHaveBeenCalled();
         });
     });
+
+    describe("Drilling in Combo chart", () => {
+        const columnPoint: Partial<IHighchartsPointObject> = {
+            ...point,
+            series: { type: VisualizationTypes.COLUMN },
+        } as any;
+
+        const linePoint: Partial<IHighchartsPointObject> = {
+            ...point,
+            x: 2,
+            y: 3,
+            series: { type: VisualizationTypes.LINE },
+            drillIntersection: [
+                {
+                    id: "id4",
+                    title: "title4",
+                    header: {
+                        identifier: "identifier4",
+                        uri: "uri4",
+                    },
+                },
+                {
+                    id: "id5",
+                    title: "title5",
+                    header: {
+                        identifier: "identifier5",
+                        uri: "uri5",
+                    },
+                },
+                {
+                    id: "id6",
+                    title: "title6",
+                    header: {
+                        identifier: "identifier6",
+                        uri: "uri6",
+                    },
+                },
+            ],
+        } as any;
```

Comment:
could you explain more about this?
`const linePoint: Partial<IHighchartsPointObject> = {...} as any;`
Does it work if we omit `as any`?

Reply:
Nope. If I omit `as any`, it will throw following error
<img width="1027" alt="Screen Shot 2019-06-20 at 1 20 52 PM" src="https://user-images.githubusercontent.com/39481271/59825556-8a6b2f80-935e-11e9-9da4-18740f3ce643.png">
Because `Series` type requires many props that we can't define all of them there :(